src/sql/azdata.d.ts

```diff
@@ -4703,6 +4703,7 @@ declare module 'azdata' {
 			output_type: OutputTypeName;
 			metadata?: {
 				azdata_chartOptions?: any;
+				resultSet?: ResultSetSummary;
```

Comment:
hmm, can we define an interface for the metadata type instead of inline definition? have you tried to add the new properties in proposed file? if possible it should be first introduced over there.

Reply:
Yep, I can defined a new interface for the metadata in the proposed file.

Reply:
Since we already have azdata_chartOptions here what I'd suggest is : 

1. Make new interface definition in azdata.d.ts that contains azdata_chartOptions
1. Then extend that in azdata.proposed.d.ts to include this new option

Doing this won't break existing users since the type is still the same